src/Projection/AttributeHandleMethod.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Projection;
+
+use Patchlevel\EventSourcing\Aggregate\AggregateChanged;
+use Patchlevel\EventSourcing\Attribute\Handle;
+use ReflectionClass;
+
+use function array_key_exists;
+
+/**
+ * @psalm-require-implements Projection
+ */
+trait AttributeHandleMethod
```

Comment:
we have a lot of traits atm tbh 😨

Reply:
:man_shrugging:

Reply:
Do you have a better idea? I think that in the next major release we can force attribute support and thereby remove all traits. But would still wait and collect ideas.

Reply:
No, no other idea atm